### PR TITLE
add @tatchi/bs-idb-keyval

### PR DESCRIPTION
**Pull request by tatchi**

### CODE REVIEW

**Comment by glennsl** [comment]
This doesn't seem to have been picked up by NPMS yet. Have you renamed it without pushing a new release afterwards? I think NPMS can miss it in that case.

File: sources.json [hunk 1]
```diff
@@ -291,6 +291,11 @@
       "platforms": ["node"],
       "keywords": ["testing"]
     },
+    "@tatchi/bs-idb-keyval": {
+      "category": "binding",
+      "platforms": ["browser"],
+      "keywords": ["idb-keyval", "indexeddb", "localstorage", "storage"]
```
Comment (glennsl) [inline sources.json:297]:
```suggestion
      "keywords": ["database"]
```

**Comment by tatchi** [comment]
Thanks for the review.

I re-published it under my namespace but without changing the version number. Didn't know npms but it's available on the official npm website.

**Comment by glennsl** [comment]
Redex depends on NPMS because it does some normalization and analytics that's useful. Could you try bumping the version to see if it gets picked up? It's also possible that their indexer is down though, as that does happen sometimes,

**Comment by tatchi** [comment]
Seems like it has finally been picked up by npms 🎉

https://npms.io/search?q=%40tatchi%2Fbs-idb-keyval

**Comment by glennsl** [comment]
Indeed it has. Thanks!

**Pull request merged**